ctags.lua

```diff
@@ -11,6 +11,9 @@ local function get_path(prefix, path)
 end
 
 local function find_tags(path)
+	if path == nil then
+		path = os.getenv('PWD') .. '/'
+	end
```

Comment:
I'd say that at this point `path` should not be `nil`. I think it would be clearer to have a separate function, say, `get_current_path`, which returns either `vis.win.file.path` or the PWD value. In this case the path will be determined early in the `tag_cmd` and `tselect_cmd` functions and no checks for `nil` will be necessary anywhere in the code.

Reply:
Done.